components/list/demo/list-item-sample.js

```diff
@@ -40,6 +42,32 @@ class ListItemSample extends ListItemMixin(ListItemCheckboxMixin(LitElement)) {
 			[slot="content-action"]:focus {
 				outline: none;
 			}
+			.d2l-list-item-drag-image {
```

Comment:
Yes, so if this is not given then it will set the image to :host. This is if you want a more special effect. I pushed the rotation of drag image to the list-item since it is visual aspect and maybe different list items would want different visualizations. 

There is also a tech reason - the mixin would need some knowledge about how the dom is setup in the list item to make the tilt work. So the above justifies this approach to fix that problem. 

I do believe in my justification since I do want the list-item components to define the style while using these mixins for their functionality and logic. I also realize, I more inclined to believe that reason has justification due to the fact there are tech reason another way won't work.